src/styles/blocks/header.scss

```
@@ -0,0 +1,12 @@
+.header {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+
+  &logo {
```

```suggestion
  &__logo {
```

if it is inside bem block, it should be a bem element